scheduler.py

```diff
@@ -1,178 +0,0 @@
-import os
-import praw
-from pymongo import MongoClient
-import yfinance as yf
-
-# Number of the most discussed stocks
-TOTAL_NUMBER_TO_PRESENT = 10
-
-# Number of the most discussed stocks we save
-# This should be greater than 10, in case Yahoo Finance is not able to provide information for certain stocks
-TOTAL_NUMBER_OF_STOCKS = 20
-
-# A list of abbreviations that generally do not represent tickers.
-TICKER_BLACKLIST = ['A', 'ALL', 'ARE', 'AT', 'CEO', 'DD', 'EDIT', 'EV', 'FOR', 'ON', 'ONE', 'OR', 'RH', 'VERY']
-
-# Information we look for in Yahoo Finance
-yahoo_finance_FIELDS = ['industry', 'previousClose', 'fiftyDayAverage', 'averageDailyVolume10Day']
-
-
-# Reads, parses and converts the ticker files into dictionary
-def build_dictionary(file_name, ticker_dict):
-    with open(file_name, 'r') as file:
-        for line in file:
-            line_list = line.strip().split('\t')
-            if len(line_list) < 2:
-                continue
-            symbol = line_list[0]
-            name = line_list[1]
-            if symbol not in ticker_dict:
-                ticker_dict[symbol] = name
-
-
-# Counts the tickers appeared in the titles and/or contents of the hot posts
-# Returns a dictionary with top 20 counts in non-increasing order
-def count_tickers(reddit, ticker_dict):
-    ticker_count = {}
-    hot_posts = reddit.subreddit('wallstreetbets').hot(limit=500)
-
-    for post in hot_posts:
-        word_in_title = post.title.split()
-        add_to_dict(word_in_title, ticker_dict, ticker_count)
-        word_in_content = post.selftext.split()
-        add_to_dict(word_in_content, ticker_dict, ticker_count)
-
-    sorted_ticker_count = dict(
-        sorted(ticker_count.items(), key=lambda item: item[1], reverse=True)[:TOTAL_NUMBER_OF_STOCKS])
-
-    return sorted_ticker_count
-
-
-# Helper function to aggregate the count of each ticker in ticker_count
-# Filters out tickers in the blacklist
-def add_to_dict(word_in_post, ticker_dict, ticker_count):
-    for word in word_in_post:
-        clean_word = word.strip()
-        if clean_word in TICKER_BLACKLIST:
-            continue
-        if clean_word in ticker_dict:
-            ticker_count[clean_word] = ticker_count.get(clean_word, 0) + 1
-
-
-# Helper function to determine if Yahoo Finance can provide the relevant information
-# If not, we skip this stock
-def has_all_finance_fields(stock_info):
-    for field in yahoo_finance_FIELDS:
-        if field not in stock_info.keys():
-            return False
-
-    return True
-
-
-# Helper function to get the most recent Top 10 stock record from the database
-# Returns a list of ticker symbols, sorted by rank
-def get_prev_stock_list(collection, current_timestamp):
-    latest_timestamp_entry = collection.find({'timeStamp': {'$lte': current_timestamp}}).sort([('timeStamp', -1)]).limit(1)
-
-    latest_time_stamp = latest_timestamp_entry[0].get('timeStamp')
-    latest_stock_obj_list = collection.find({'timeStamp': latest_time_stamp})
```

Comment:
collection.find({'timeStamp': latest_time_stamp}).sort([('rank', 1)])

this is not working?